### ISSUE TITLE
Removal of some tags

### ISSUE THREAD

**Issue by Regulator13** [famegames/squared#124]
As  @Phabbits pointed out, if we are allowing our contributors to help as they see best, perhaps it is premature to label something as sprite change when it could be handled in a different manner.

That being said I think we should remove the `change object` and `change sprite` labels, especially since `change object` applies to almost every issues. Note the `animation` label will still stay.

I think the `change room` and `change script` labels can stay since they are more infrequent and actually have bearing on who works on things.

**Comment by Phabbits** [comment]
I agree that with our current direction of how issues will be submitted, it is appropriate to remove these labels. I even suggest to remove the change script label, since there are many ways to avoid using a script, although rarely a good decision, and if the assignee decides a script will be used that might interfere with others, they can specify so in a comment.
